### ISSUE TITLE
please re-check summary of responses by continent

### ISSUE THREAD

**Issue by JackDougherty** [HandsOnDataViz/survey-results#1]
@ilyankou - I made the repo public, added more info to the README, and tried to reconstruct a table of responses by continent based on your analysis of the data. But the numbers don't add up to what you wrote at the top of the map. Would you please re-check your notes and update this? 
https://github.com/HandsOnDataViz/survey-results/blob/main/data/responses-by-continent.csv
Thanks

**Comment by ilyankou** [comment]
Apologies for the inaccuracies. See updated map in the repo.

**Comment by JackDougherty** [comment]
thanks @ilyankou for clearing up my confusion. FYI, I moved your excellent "Null Island" comment from the dataviz into a full paragraph at the bottom of the [Bad Data section of chapter 3](http://handsondataviz.org/bad-data.html). Please also see non-urgent request for better maps there.

Please check how I updated the title & description at the top of the chart and map in the README. Let me know if I understand the chart when I write this in the book in chapter 16: "In particular, 89 percent reported completing the equivalent of a college degree (16 or more years of schooling), but rated themselves at the lower end of the experiential scale, as shown in the right side of Figure..." [TODO: ILYA please clarify if 89% are higher ed AND lower scale of experience, or if those are separate.] 

When you have time, would you please look into two things? This is NOT urgent and can be dealt with during revisions phase:

1. Now that our book says to beware of poorly geocoded data, let's make sure we're following our own advice. Please confirm if Null Island and other weird results have been removed from this CSV, and update it if needed, so that our interpretation of the map is correct. 
https://github.com/HandsOnDataViz/survey-results/blob/main/data/responses-by-continent.csv

2. When inspecting the geocoded survey response data file, I was surprised that around 1 out of 12 had no continent listed, despite the presence of good-looking location data, but maybe incorrect geocoding? For example, see row 8 Nova Scotia and row 21 Barbados in the screenshot. Are you confident in Nominatum's geocoding? If not, do we have other automated options? Or should we manually extract the non-geocoded results and run them through Google Sheets geocoder?
![geocoding-quality-issue](https://user-images.githubusercontent.com/649719/97625806-606c5780-19ff-11eb-9e6a-6926caccabae.png)

thanks

**Comment by ilyankou** [comment]
@JackDougherty 

1 - I removed all null-geocoded points from the map.

2 - I used [this continents geojson file](https://gist.github.com/hrbrmstr/91ea5cc9474286c72838) to determine continents based on points. If you zoom in on Barbados and Halifax, you will see where the problem lies. To fix this, I added a 0.5 degree buffer to all continent boundaries to make sure islands/edges are covered, and I then manually fixed those that still fell outside the boundaries. Because this is an important step where points may get missing, I added an extra row to the summary file, called *geocoded but falling outside of continents polygons*. Currently they are set to 0 as I was able to manually fix them.

<img width="482" alt="barbados" src="https://user-images.githubusercontent.com/10443203/97689723-eafe9680-1a93-11eb-9b49-93bf2f8d0ef9.png">
<img width="516" alt="halifax" src="https://user-images.githubusercontent.com/10443203/97689726-eb972d00-1a93-11eb-9e91-6576e2cd6ff0.png">
